samples/NEO/HelloWorld.neo

```diff
@@ -0,0 +1 @@
+echo("Hello, world")
```

Comment:
We prefer to avoid trivial sample files because they are then used to classify files in the Bayesian classifier. Please instead use example programs from "real" applications under permissive licenses.

This comment applies to all current sample files.

Reply:
ok